app.js

```diff
@@ -0,0 +1,24 @@
+
+/**
+ * Dependencies
+ */
+
+var debug = require('debug')('app');
```

Comment:
I like the debug info, but do we have different logging levels?
Before we had a logging class, where you can configure different levels and get output depending on what kind of environment you were. I liked that, but I'm sure you can convince me to use just debug.